core/src/test/java/kafka/test/junit/ClusterTestExtensionsUnitTest.java

```diff
@@ -17,29 +17,66 @@
 
 package kafka.test.junit;
 
+import kafka.test.ClusterConfig;
 import kafka.test.annotation.ClusterTemplate;
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class ClusterTestExtensionsUnitTest {
+
+    static class StubTest {
```

Comment:
It seems this nested class is unnecessary. For example:

```java
    static List<ClusterConfig> cfgEmpty() {
        return new ArrayList<ClusterConfig>();
    }

    @SuppressWarnings({"unchecked", "rawtypes"})
    private ExtensionContext buildExtensionContext(String methodName) throws Exception {
        ExtensionContext extensionContext = mock(ExtensionContext.class);
        Class clazz = ClusterTestExtensionsUnitTest.class;
        Method method = clazz.getDeclaredMethod(methodName);
        when(extensionContext.getRequiredTestClass()).thenReturn(clazz);
        when(extensionContext.getRequiredTestMethod()).thenReturn(method);
        return extensionContext;
    }
```

BTW, `methodName` is always "cfgEmpty", so maybe we can remove it from input argument.

Reply:
In my mind, keeping the argument will make this function reusable if needed.